src/data/equiv/basic.lean

```diff
@@ -240,6 +240,38 @@ theorem left_inverse_symm (f : equiv α β) : left_inverse f.symm f := f.left_in
 
 theorem right_inverse_symm (f : equiv α β) : function.right_inverse f.symm f := f.right_inv
 
+@[simp] lemma injective_comp (e : α ≃ β) (f : β → γ) : injective (f ∘ e) ↔ injective f :=
```

Comment:
Can we generalize the forward direction here to `function.injective.of_comp_bijective`? There's no need to have an equiv here, bijectivity should be enough I think?